terra-workspace-manager/sa.tf

```diff
@@ -18,10 +18,10 @@ resource "google_project_iam_member" "sqlproxy" {
 
 locals {
   app_sa_roles = [
-    # Allow exporting metrics, profiling, and tracing for monitoring.
-    "roles/monitoring.editor",
     "roles/cloudprofiler.agent",
     "roles/cloudtrace.agent",
+    "roles/monitoring.editor", # Allow exporting metrics, profiling, and tracing for monitoring.
```

Comment:
nit: I wouldn't expect an inline comment to also apply to the two previous lines. Can also split this comment so that the cloudprofiler.agent is for profiling, cloudtrace for tracing, monitoring for metrics.

Reply:
Good point.